.nuspec/Microsoft.Maui.Controls.targets

```diff
@@ -25,6 +25,7 @@
 
 	<ItemGroup Condition="'$(AndroidEnableProfiledAot)' == 'true' and '$(MauiUseDefaultAotProfile)' != 'false'">
 		<AndroidAotProfile Include="$(MSBuildThisFileDirectory)maui.aotprofile" />
+		<AndroidAotProfile Include="$(MSBuildThisFileDirectory)maui-blazor.aotprofile" Condition=" '$(UsingMicrosoftNETSdkRazor)' == 'true' " />
```

Comment:
Does it hurt to just always use both profiles? Would there be any advantage to doing that?

Reply:
It seems like that wouldn't hurt, I see some common stuff like `TaskCompletionSource` and `System.Text.Json` in the `maui-blazor` profile.

Reply:
I can easily see non-Blazor apps using that code.